docker-runner/docker-run.sh

```diff
@@ -7,4 +7,11 @@
 : "${CONTAINER_NAME:?}"
 
 set -e
-docker run --rm --link $CONTAINER_NAME:target -e ZGRAB_TARGET=target zgrab2_runner $@
+
+MOUNT=
+if [ -n "$MOUNT_HOST" ]; then
```

Comment:
I feel like if we are going to give the mount special treatment (rather than having an opaque e.g. `EXTRA_DOCKER_ARGS`), we should be more opinionated; like maybe only let them set the working directory instead of explicitly taking both the host and container dirs (e.g. `-v "${DOCKER_INPUT_DIR:-"$(realpath .)"}":/var/tmp/scan-root`, and then update the entrypoint.sh to `cd /var/tmp/scan-root` before running zgrab2; then you could even have default behavior that mounts the host cwd as the container cwd, so that relative paths remain the same...